pyproject.toml

```diff
@@ -70,6 +70,7 @@ requests = { version = "*", optional = true }
 
 # [tool.poetry.group.other.dependencies]
 urllib3 = { version = "^1.25", optional = true }
+Bottleneck = { version = "*", optional = true }
```

Comment:
Why does it start with a capital letter?

Reply:
https://pypi.org/project/Bottleneck/